music_assistant/server/providers/deezer/__init__.py

```diff
@@ -530,10 +528,11 @@ def parse_track(
         extra_init_kwargs: dict[str, Any] | None = None,
     ) -> Track | PlaylistTrack:
         """Parse the deezer-python track to a MASS track."""
-        if "position" in extra_init_kwargs:
```

Comment:
this can be simplified to a default initializer:

if extra_init_kwargs is None:
    extra_init_kwargs = {}